### DISCUSSION TITLE
Vipsthumbnail fails for png: profile incompatible with image

### DISCUSSION THREAD

**Discussion by le0daniel** [libvips/libvips#2132] [Q&A]
## Bug report

**Describe the bug**
When using following command
```
/usr/bin/vipsthumbnail image.png --size 4096x\> -e srgb --output 'output.png[strip]'
```
on this image: 
![a2de6c2b0eb4a33e854bf7daf14e1e68](https://user-images.githubusercontent.com/11374766/109619272-deb94180-7b38-11eb-89f0-367ccfc19997.png)

I get following output:
```
(vipsthumbnail:593): VIPS-WARNING **: 08:18:22.903: profile incompatible with image
/usr/bin/vipsthumbnail: unable to thumbnail image.pg
icc_transform: no input profile
```

Vips Version: `vips-8.10.5-Fri Dec 18 10:14:09 UTC 2020`

I m not sure to be honest if this is a broken image or a bug. At least OSX Preview shows the image.

**To Reproduce**
1. Use the command above to generate a thumbnail. 

**Expected behavior**
Converts the image successfully

**Actual behavior**
Fails to connvert the image

**Environment**
 - OS: Amazon linux v2
 - Vips: 8.10.5

**Comment by jcupitt** [comment]
Hi @le0daniel,

This is an RGBA PNG with a CMYK profile attached to it. If you give the `-e srgb` option, `vipsththumbnail` will try to convert the input to srgb using the bad profile. This then fails, obviously.

It should work without the `-e` option. I see:

```
$ vipsthumbnail amgen.png  --size 4096x\> -e srgb --output output.png[strip]
(vipsthumbnail:253989): VIPS-WARNING **: 10:25:39.057: iCCP: profile 'icc': 'CMYK': invalid ICC profile color space
...
$ vipsheader output.png 
output.png: 590x139 uchar, 4 bands, srgb, pngload
```

**Comment by jcupitt** [comment]
I moved this to a Q&A discussion since I don't think it's a libvips bug -- I would expect this command to fail.

Any ideas for improving libvips behaviour are very welcome, of course.
